masterlist.yaml

```diff
@@ -28854,16 +28854,14 @@ plugins:
       - <<: *alreadyInX
         subs: [ 'Immersive Weapons_CCOR_Patch.esp' ]
         condition: 'active("Immersive Weapons_CCOR_Patch.esp")'
-  - name: '(imp_helm_legend_WACCF_Patch|ImprovedClosefacedHelmets_WACCF)\.esp'
-    msg:
-      - <<: *alreadyInX
-        subs: [ 'imp_helm_legend_WACCF_AMB_Patch.esp' ]
-        condition: 'active("imp_helm_legend_WACCF_AMB_Patch.esp")'
-      - <<: *useInstead
-        subs: [ 'imp_helm_legend_WACCF_AMB_Patch.esp' ]
-        condition: 'active("aMidianBorn_ContentAddon.esp") and version("aMidianBorn_ContentAddon.esp", "2.0", >=)'
   - name: 'ImprovedClosefacedHelmets_WACCF.esp'
     msg: [ *obsolete ]
+  - name: 'imp_helm_legend_WACCF_AMB_Patch.esp'
+    msg:
+      - <<: *obsolete
```

Comment:
If you write
```yaml
      - <<: *obsolete
```
then either a `subs` or `condition`, or the combination of those two should follow:
```yaml
  - name: 'plugin.esp'
    msg:
      - <<: *obsolete
        subs: [ 'somesub' ]
        condition: 'somecondition'
```
If there's no `subs` or `condition` you want to use, then you could write:
```yaml
    msg:
      - *obsolete
      - <<: *useInstead
        subs: [ 'imp_helm_legend_waccf_patch.esp and imp_helm_legend_amb_patch.esp' ]
        condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp") and active("aMidianBorn_ContentAddon.esp") and version("aMidianBorn_ContentAddon.esp", "3.0", >=)'
```